File: sea-shell-service-spring-web-flux/sea-shell-data/src/main/java/org/jesperancinha/shell/webflux/data/SeaShellAccountDto.java
package org.jesperancinha.shell.webflux.data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.math.BigDecimal;

@Builder
@Getter
@Setter
@EqualsAndHashCode
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class SeaShellAccountDto {
    private BigDecimal value;
    private String currency;
}

File: sea-shell-service-spring-web-flux/src/main/kotlin/org/jesperancinha/shell/webflux/data/SeaShellAccountDto.kt
package org.jesperancinha.shell.webflux.data

import lombok.*
import java.math.BigDecimal

@Builder
@Getter
@Setter
@EqualsAndHashCode
@ToString
@AllArgsConstructor
@NoArgsConstructor
class SeaShellAccountDto {
    private val value: BigDecimal? = null
    private val currency: String? = null
}